<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DS_Lop</class>
 <widget class="QDialog" name="DS_Lop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Quay lại</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>691</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblMaLop">
        <property name="text">
         <string>Mã Lớp:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editMALOP"/>
      </item>
      <item>
       <widget class="QLabel" name="lblTenLop">
        <property name="text">
         <string>Tên Lớp:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editTENLOP"/>
      </item>
      <item>
       <widget class="QLabel" name="lblNamHoc">
        <property name="text">
         <string>Năm học:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editNAMHOC"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnThem">
        <property name="text">
         <string>Thêm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSua">
        <property name="text">
         <string>Sửa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnXoa">
        <property name="text">
         <string>Xóa</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableLop">
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>229</number>
      </attribute>
      <column>
       <property name="text">
        <string>Mã lớp </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tên lớp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Năm học </string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Danh Sách Lớp </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="margin">
    <number>-3</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
